ui/.eslintrc.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     'ember/no-actions-hash': 'off',
     'ember/require-tagless-components': 'off',
     'ember/no-component-lifecycle-hooks': 'off',
+    'ember/use-ember-data-rfc-395-imports': 'off',
```

Comment:
I installed the ember data types package from DefinitelyTyped and it's namespaced to `ember-data` rather than `@ember-data`. Since we're about to remove Ember Data I turned this rule off so the types could be imported where needed.

Reply:
Turned it back on after making some further updates 😁